src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -220,8 +224,8 @@ def get_aggregate_prediction_intervals(
         aggregate_prediction_intervals = (
             last_election.merge(modeled_bounds, how="inner", on=aggregate)
             .assign(
-                predicted_lower=lambda x: np.maximum(x[f"last_election_results_{estimand}"] + x.lb, 0),
-                predicted_upper=lambda x: np.maximum(x[f"last_election_results_{estimand}"] + x.ub, 0),
+                predicted_lower=lambda x: np.maximum(x[f"last_election_results_{estimand}"] + x.lb, aggregate_nonreporting_votes[f"results_{estimand}"]),
```

Comment:
can you add a comment above this explaining why this is necessary. something around the lines of, the gaussian model re-uses the unadjusted bounds to estimate the gaussian for the aggregates. so we need to do this maximum again.

Reply:
Added